common/src/main/java/net/neoforged/gradle/common/runtime/tasks/SourceAccessTransformer.java

```diff
@@ -38,6 +33,15 @@ public SourceAccessTransformer() {
 
                             args.add("--libraries-list=" + getLibraries().get().getAsFile().getAbsolutePath());
 
+                            final StringBuilder builder = new StringBuilder();
+                            getClasspath().forEach(f -> {
+                                if (!builder.isEmpty()) {
+                                    builder.append(File.pathSeparator);
+                                }
+                                builder.append(f.getAbsolutePath());
+                            });
+                            args.add("--classpath=" + builder.toString());
```

Comment:
Do we think the classpath will ever become so large as to exceed OS limitations on command-line length (if any)?

Reply:
Maybe, I have plans to fix this though :dagger: 
Just not in a PR that is a fix for an issue.